packages/amplication-data-service-generator/src/util/ast.ts

```diff
@@ -769,3 +779,78 @@ export function createGenericArray(
     builders.tsTypeParameterInstantiation([itemType])
   );
 }
+
+export function removeDecoratorByName(
```

Comment:
```suggestion
// This function removes all instances of a decorator by its name. in case the same decorator exists multiple times, it will be removed from all locations
export function removeDecoratorByName(
```

Reply:
added the comment